public/scss/login_register.scss

```diff
@@ -0,0 +1,63 @@
+.login,
+.register {
+  .login__form {
+    padding: 70px 0 20px;
+    p {
+      font-size: 16px;
+    }
+    .form_text {
+      padding: 0 60px 0 110px;
+    }
+    .form_set {
+      max-width: 400px;
+      width: 100%;
+    }
+  }
+}
+
+
+@media (max-width: 1023px) {
+  .login,
+  .register {
+    .login__form {
+      max-width: 900px;
+      margin: 0 auto;
+      .form_text {
+        padding: 0 15px 0 0;
+      }
+    }
+    .flex_box {
+      -ms-flex-wrap: wrap;
+      flex-wrap: wrap;
+      & > div {
+        -webkit-box-flex: 0;
+        -ms-flex: 0 1 50%;
+        flex: 0 1 50%;
+      }
+    }
+  }
+}
+
+@media (max-width: 479px) {
+  .login,
+  .register {
+    .login__form {
+      padding: 40px 0 20px;
+      .form_text {
+        padding: 0 0px 30px 0;
+      }
+      .form_set {
+        max-width: 100%;
+      }
+    }
+    .flex_box {
```

Comment:
Don't like these overriding of generic classes. They should only be written at one place and used as is. Overwriting should happen with-in the module for module specific context classes

Reply:
Done